### PR TITLE
Added default configuration parameter for Config

### LINKED ISSUE #126: Config – an easy way to load the default config
As posted here - https://github.com/ckeditor/ckeditor5-core/pull/72#issuecomment-283611499 - we need to introduce default parameters for `Config`.

It is required for support `Editor.build` property (https://github.com/ckeditor/ckeditor5-core/issues/67).

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Added default configuration parameter for `Config`. Closes #126.

---

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -31,6 +31,11 @@ export default class Config {
 		if ( configurations ) {
 			this._setObjectToTarget( this._config, configurations );
 		}
+
+		// Set default configuration.
+		if ( defaultConfigurations ) {
+			this._setObjectToTarget( this._config, defaultConfigurations, true );
```
Comment (Reinmar) [inline src/config.js]: Why not `define()`? It's a role of this method to set the default values.

Comment (pomek) [inline src/config.js]: Fixed in `9bd0d4b`.

**Comment by Reinmar** [comment]
Please add a test for a deeper config objects.

**Comment by pomek** [comment]
Done in 4783f38.

File: tests/config.js [hunk 2]
```diff
@@ -41,9 +41,75 @@ describe( 'Config', () => {
...
+			expect( config.get( 'bar' ) ).to.equal( 2 );
+		} );
+
+		it( 'should work with deeper objects', () => {
+			const defaultConfig = {
+				a: {
+					first: 1,
+					second: 2
+				},
+				'b.foo.first': 1,
```
Comment (Reinmar) [inline tests/config.js]: Not like this, like:

```js
b: {
   foo: {
       first: 1
   }
}
```

This is supported just by coincident :D

Comment (pomek) [inline tests/config.js]: Fixed in 2e7eab7. :D

**Comment by Reinmar** [comment]
You forgot about API docs but I'll add them. R+ for the rest.

**Pull request merged**